ProjectSettings/ProjectVersion.txt

```diff
@@ -1,2 +1,2 @@
-m_EditorVersion: 5.4.0b14-HTP
+m_EditorVersion: 5.4.0b18-HTP
```

Comment:
How do we feel about keeping the project up to date with the Unity drops available? I'm all for anything that pushes this number higher, but it should probably be in a separate pull right?